CPAC/alff/alff.py

```diff
@@ -314,9 +353,10 @@ def alff_falff_space_template(wf, cfg, strat_pool, pipe_num, opt=None):
     node, out = strat_pool.get_data(["space-template_res-derivative_desc-bold_mask",
                                      "space-template_desc-bold_mask"])
     wf.connect(node, out, alff, 'inputspec.rest_mask')
-    outputs = {
+    
+    outputs = outputs.update({
```

Comment:
`dict().update` works in-place
```suggestion
    outputs.update({
```